src/main/java/seedu/address/logic/commands/LinkCommand.java

```diff
@@ -0,0 +1,144 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INTERNSHIP;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PERSON;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_INTERNSHIPS;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.internship.Internship;
+import seedu.address.model.person.Person;
+
+/**
+ * Links a Person and an Internship.
+ */
+public class LinkCommand extends Command {
+
+    public static final String COMMAND_WORD = "link";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Links a Person and an Internship. "
+            + "Parameters: "
+            + PREFIX_PERSON + "PERSON_INDEX "
+            + PREFIX_INTERNSHIP + "INTERNSHIP_ID "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_PERSON + "0 "
+            + PREFIX_INTERNSHIP + "0 ";
+    
```

Comment:
CI claimed that there is trailing whitespace here, might want to remove it so it passes